### ISSUE TITLE
Addin additional String helpers

### ISSUE THREAD

**Issue by Andras-Csanyi** [Handlebars-Net/Handlebars.Net.Helpers#74]
Hi All,

I'm using this package and I like it very much, however, I miss the String.Substring helper. Should I feel free to open a pull request containing this helper? Is there anything I need to be aware of which is contribution related beside the usual things? Usual things: clone the repo, create your branch, make your changes, test them and open the PR, etc...

**Comment by StefH** [comment]
@Andras-Csanyi 
Yes you can add this, also add a unit test to test the helper itself and to test the helper in a template.

**Comment by StefH** [comment]
@Andras-Csanyi 
If you have time, you can copy here the markdown for the changes for the wiki page:
https://github.com/Handlebars-Net/Handlebars.Net.Helpers/wiki/String

**Comment by Andras-Csanyi** [comment]
> @Andras-Csanyi If you have time, you can copy here the markdown for the changes for the wiki page: https://github.com/Handlebars-Net/Handlebars.Net.Helpers/wiki/String

Ok, I'll do it later today. Thanks!
